spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Recurrent.scala

```diff
@@ -264,6 +265,16 @@ class Recurrent[T : ClassTag]()
     output
   }
 
+  def getFinalstateAndCell(): mutable.HashMap[String, Tensor[T]] = {
+    require(cells != null && cells(times - 1).output != null,
+      "getFinalstateAndCell need to be called after updateOutput")
+    val map = new mutable.HashMap[String, Tensor[T]]()
+    map("cell") = cells(times - 1).output.toTable(hidDim).asInstanceOf[Table]
```

Comment:
For SimpleRNN, the cells(times-1).output.toTable(hidDim) is a Tensor. Thus, it will throw an error.